runtime/android/core_internal/src/org/xwalk/core/internal/XWalkContent.java

```diff
@@ -132,11 +136,24 @@ private void setNativeContent(long newNativeContent) {
 
         assert mNativeContent == 0 && mCleanupReference == null && mContentViewCore == null;
 
+        boolean animated;
```

Comment:
Since this is also related to initializing `ContentViewRenderView`, it makes sense to move it after the comment.